LayoutTests/fast/harness/results.html

```diff
@@ -331,9 +331,9 @@
         let extensionIndex = buffer.lastIndexOf(".");
         let filename = buffer.substr(0, extensionIndex);
         // Test names that are acutally process names are treated like they don't have any extension
```

Comment:
Nit: Fix "acutally" to "actually" while we're here.

```suggestion
        // Test names that are actually process names are treated like they don't have any extension.
```